### ISSUE TITLE
User public IP is not Changing to netbird IP

### ISSUE THREAD

**Issue by nouman1657** [netbirdio/netbird#1863]
Hi All,

Newbie here for Netbird.

We are trying to implement Netbird in our environment and we love its features for connecting devices with each other. However, we have some portals that are restricted to specific public IPs. When a user is working from home and connects to the Netbird VPN, their IP does not change to a static Netbird IP. As a result, we are unable to access those portals. We are using Netbird self-hosted version 0.27.3. Can anyone suggest if Netbird offers SSL VPN features that allow changing my public IP to the server's public IP?

**Comment by alexmoras** [comment]
You would need to configure Netbird to route all traffic to a specific peer. By default, Netbird only routes traffic targeted for another Netbird peer over the VPN. Everything else uses the default routes specified by your OS.

They've got a guide on how to route all internet traffic to a single peer, which would then cause it to have the external IP of the routing peer.

https://docs.netbird.io/how-to/configuring-default-routes-for-internet-traffic

**Comment by nouman1657** [comment]
Yeah I have followed the same document but still my "Add Exit Node" button is missing on dashboard. Any suggestions?

**Comment by Zain0004** [comment]
I am facing the same issue in my environment, The "Add Exit Node" button is missing. Please HELP.

**Comment by lixmal** [comment]
@nouman1657 @Zain0004 Are you running self-hosted? You might need to update the dashboard version, this was added recently.

Alternatively you can just route `0.0.0.0/0`, same effect

**Comment by Zain0004** [comment]
I am a newbie to netbird. Can you please tell me how to update the **dashboard version** as i have already installed the latest version of netbird.

**Comment by taylorwilsdon** [comment]
The exit node management feature on the dashboard wasn't available in the docker image for netbirdio/dashboard:main until a few hours ago, but it is now - you can pin the `[v2.3.0]` tag to get it!

**Comment by ukashazia** [comment]
> You would need to configure Netbird to route all traffic to a specific peer. By default, Netbird only routes traffic targeted for another Netbird peer over the VPN. Everything else uses the default routes specified by your OS.
> 
> They've got a guide on how to route all internet traffic to a single peer, which would then cause it to have the external IP of the routing peer.
> 
> https://docs.netbird.io/how-to/configuring-default-routes-for-internet-traffic

I followed the steps, setup my VPS as exit node for a desktop and android device in the same "peer group". But still my external/public Ip of peripheral devices (android and desktop) is not that of the public Ip of my VPS. Am I doing something wrong or missing something?

**Comment by nazarewk** [comment]
Do you still have the issue with using Exit Nodes on latest NetBird version?

**Comment by YtachY** [comment]
I try cloud server successfully. Thanks for the [answer](https://github.com/netbirdio/netbird/issues/1863#issuecomment-2065188904) above.

The setup is completely simple. I followed exact this document and added "Distribution group" = "All" > so all of my peers will  have Exit Node's IP (They have their own IPxLocation before adding exit node).

> https://docs.netbird.io/how-to/configuring-default-routes-for-internet-traffic

---
Just in case: I have my own setup which added 1 more authentication layer: I have to **_approve_** the exit node before it connect other peers, followed [this instruction.](https://docs.netbird.io/how-to/approve-peers). Otherwise the Exit Node does not apply routing traffics.

I levaraged [free tier of GCP](https://cloud.google.com/free/docs/free-cloud-features#free-tier-usage-limits) for daily use Exit Node.

**Comment by PizzaLovingNerd** [comment]
Closing as implemented. NetBird supports this with Exit Nodes, which route internet traffic through a selected peer so clients use that peer’s public IP. See https://docs.netbird.io/manage/network-routes/use-cases/exit-nodes. Cheers!
